web/styles/zulip.css

```diff
@@ -636,6 +636,16 @@ body.has-overlay-scrollbar {
     }
 }
 
+#popover-overlay {
```

Comment:
Let's add a brief comment at this:

/* Styles for popovers rendered as an overlay at the center of the screen, primarily used in mobile sizes. */

Reply:
Added, thanks!